pamelalozano/src/main/webapp/index.html

```diff
@@ -88,7 +87,7 @@ <h4>Press &amp Charge</h4>
     </div>
     </div>
     
-<script src="https://code.jquery.com/jquery-3.2.1.slim.min.js" integrity="sha384-KJ3o2DKtIkvYIK3UENzmM7KCkRr/rE9/Qpg6aAZGJwFDMVNA/GpGFF93hXpG5KkN" crossorigin="anonymous"></script>
+<script src="script.js"></script>
```

Comment:
This should be the last script before \</body> in case you need to reference anything from popper or bootstrap. I don't think it will make a difference here, but it's a good practice.

Reply:
Done! Thanks